### PR TITLE
Introduce /test-all and /ok-to-test bot commands to documentation

### PR DESCRIPTION
**Pull request by crimsonfaith91**
Refer: https://github.com/kubernetes/test-infra/pull/2891

### CODE REVIEW

**Comment by grodrigues3** [comment]
Err, the comment (like the one in this pr) still says:

`@k8s-bot ok to test`

**Comment by crimsonfaith91** [comment]
@grodrigues3 Currently both old and new commands work. should i change the munger in another PR?

**Comment by crimsonfaith91** [comment]
@grodrigues3 introduce the new `/test all` and `/ok-to-test` bot commands to all documentations in test-infra directory.

**Comment by grodrigues3** [comment]
/lgtm

please squash and i can merge

**Comment by krzyzacy** [comment]
not really :-(

**Comment by crimsonfaith91** [comment]
no joke given! i will investigate this!

**Comment by crimsonfaith91** [comment]
@k8s-bot test this

File: prow/plugins/trigger/pr.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func handlePR(c client, pr github.PullRequestEvent) error {
 func askToJoin(ghc githubClient, pr github.PullRequest) error {
 	commentTemplate := `Hi @%s. Thanks for your PR.
 
-I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that this patch is reasonable to test. If it is, they should reply with ` + "`@k8s-bot ok to test`" + ` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should join the org to skip this step.
+I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that this patch is reasonable to test. If it is, they should reply with ` + "`/ok-to-test` or `@k8s-bot ok to test`" + ` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should join the org to skip this step.
```
Comment (spxtr) [inline prow/plugins/trigger/pr.go]: Only show the new version, not both.

Comment (crimsonfaith91) [inline prow/plugins/trigger/pr.go]: For now, we need to have old and new commands work. After a few weeks, we can retire the old commands. i will have another PR updating the documentation.

**Comment by crimsonfaith91** [comment]
Will it be fine for someone to `@k8s-bot ok to test` this PR? it seems that the new `/ok-to-test` does not work yet as i haven't redeployed the changes to prow...

**Comment by cblecker** [comment]
@k8s-bot ok to test

File: prow/plugins/trigger/pr.go (hunk 1 above)
Comment (cblecker) [inline prow/plugins/trigger/pr.go]: But the point is we should redirect users to the *new* commands now, even though the old ones will still technically work

Comment (crimsonfaith91) [inline prow/plugins/trigger/pr.go]: noted! i will remove all the old commands now.

**Comment by crimsonfaith91** [comment]
updated the docs. should be good to merged now. :)

**Comment by crimsonfaith91** [comment]
Thanks! @rmmh has already done so: https://github.com/kubernetes/test-infra/commit/cf1ac2f67142e91cb36978691e1718c007a3377a

**Comment by cblecker** [comment]
@crimsonfaith91 But you're making *another* change to prow: https://github.com/kubernetes/test-infra/pull/3106/files#diff-724ad4c6df2c629f3e0d0da9f8caf111R79

**Comment by crimsonfaith91** [comment]
You are right! I will do it this time after it gets merged.

**Pull request merged**